ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -66,12 +66,24 @@
 #define DBG FALSE
 #endif
 
+/**
+ * @brief Possible tensorflow-lite delegates.
+ */
+typedef enum {
+  TFLITE_DELEGATE_NONE = 0,
+  TFLITE_DELEGATE_GPU,
+  TFLITE_DELEGATE_NNAPI,
+
```

Comment:
Q) Is there any intention for this empty line?

Reply:
Oh it's for the new delegates, XNNPACK